### DISCUSSION TITLE
lora微调后使用api_sever调用并无效果，inference_hf.py调用却有效果

### DISCUSSION THREAD

**Discussion by hzyliusha** [zai-org/ChatGLM3#1045] [Q&A] [answered]
有没有人微调过chatglm3后调用api，为什么我微调后直接使用inference_hf.py和使用api_sever.py调用相差很大，api_sever可以说基本没有微调一样，api_sever.py已经换了inference.py的调用方式
![屏幕截图 2024-03-28 005302](https://github.com/THUDM/ChatGLM3/assets/88577911/f5fafae8-746c-4728-819d-ecbd9e7ae5f5)

**Comment by Jackiemin233** [comment]
同个问题 inference效果很好 ，但是一用api就开始抽搐

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
所有的非inference的内容调用模型部分都要修改成inference的方式，这个是要改的，不是直接套的
